terraform/shared/shared.tf.json.template.py

```diff
@@ -403,28 +406,28 @@ def paren(s: str) -> str:
                         })
                     }
                 },
-                'aws_cloudwatch_event_bus': {
-                    'securityhub': {
-                        'name': 'securityhub'
-                    }
-                },
                 'aws_cloudwatch_event_rule': {
                     'inspector': {
                         'name': 'inspector',
-                        'event_bus_name': '${aws_cloudwatch_event_bus.securityhub.arn}',
                         'event_pattern': json.dumps({
-                            'source': ['aws.securityhub'],
-                            'detail-type': ['Security Hub Findings - Imported'],
-                            'detail.ProductArn': ['arn:aws:securityhub:us-east-1::product/aws/inspector'],
-                            'detail.Severity.Label': ['CRITICAL', 'HIGH']
+                            'source': ['aws.inspector2'],
+                            'detail-type': ['Inspector2 Finding'],
+                            'detail.severity': ['CRITICAL', 'HIGH'],
+                            'detail.status': ['ACTIVE']
                         })
                     }
                 },
                 'aws_cloudwatch_event_target': {
                     'inspector_to_sns': {
-                        'event_bus_name': '${aws_cloudwatch_event_bus.securityhub.arn}',
                         'rule': '${aws_cloudwatch_event_rule.inspector.name}',
-                        'arn': '${aws_sns_topic.monitoring.arn}'
+                        'arn': '${aws_sns_topic.monitoring.arn}',
+                        'input_transformer': {
+                            'input_template': dedent(f'''\
```

Comment:
This is now how we embed JSON.

Reply:
The input template is not JSON, rather needs to be JSON after EventBridge replaces any keywords. I've added a comment above this section explain why the uncommon syntax is being used here.